src/Smidge.Core/CompositeFiles/DefaultUrlManager.cs

```diff
@@ -171,4 +173,4 @@ private string GetCompositeUrl(string fileKey, string fileExtension, string cach
                     cacheBusterValue));
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
Undo.

Reply:
Removed the new line on 176. Had to do this outside of Visual Studio.